web/backend/app.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint
```

Comment:
all of your files to be installed should be in files/* probably for you: `files/web/backend/` and `files/web/backend`

An "installed file" is to be placed outside the repository context. so your app.py probably should get installed to: `/etc/hobby-hub/web/backend/app.py` from it's repository location: `~/the-hobby-hub/files/web/backend/appy.py`. you can recursively copy a directory so it shouldn't be that hard to copy all of your files.

this issues goes with all of the files in the current "web" directory.